week-05-202402/Headers/Exercise2/Estudiante.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include "sstream"
+#include "Persona.h"
+
+class Estudiante : public Persona
+{
+protected:
+	string DNI;
+	string nivelEstudios;
+
+public:
+	Estudiante(string nombre, string apellido, int edad, char sexo, string nivelEstudios);
+	string generarDNI();
+};
+
+Estudiante::Estudiante(string nombre, string apellido, int edad, char sexo, string nivelEstudios) : Persona(nombre, apellido, edad, sexo)
+{
+	this->nivelEstudios = nivelEstudios;
+	DNI = generarDNI();
+}
+
+string Estudiante::generarDNI()
+{
+	stringstream ss;
+	for (int i = 0; i < 8; i++)
+	{
+		ss << rand() % 10;
+	}
```

Comment:
We could use:
string s;
s = to_string(rand() % (99999999 - 10000000 + 1) + 10000000